### DISCUSSION TITLE
Return / Exit Code for Warning?

### DISCUSSION THREAD

**Discussion by HaKvi** [amidaware/tacticalrmm#1586] [Q&A] [answered]
Hi All! I'm writing some check scripts. They are all working perfectly, almost. I only know the the 2 return codes 0 (success) and 1 (error). In PS script theis means "exit 0" or "exit 1". Is there a return code for a warning (yellow)? The result of the scripts are not worth to mark a server in red. THX

**Comment by silversword411** [comment, marked as answer]
Checks only support `i`, pass, and fail atm

If you want warning, use a task

**Reply by HaKvi** [reply]
Or mark the error return value (=1) as an information or warning. This is working for me thx
